### ISSUE TITLE
branding: distro logo is placed behind dark overlay in dark mode

### ISSUE THREAD

**Issue by logonoff** [cockpit-project/cockpit#21378]
### Explain what happens

In the login page, the distro icon appears behind the darkened layer by mistake

![image](https://github.com/user-attachments/assets/ed674dc5-40ca-47f3-8116-b5ebd1081074)
![image](https://github.com/user-attachments/assets/40491eda-ce64-4bf0-b868-583bd7270130)
![image](https://github.com/user-attachments/assets/a1cea5a5-f252-4865-862f-cbf60d6fd1f4)
![image](https://github.com/user-attachments/assets/b52cdc72-919a-46dc-ba79-192404b1efa4)

### Version of Cockpit

cockpit 323.1

### Where is the problem in Cockpit?

Unknown or not applicable

### Server operating system

Red Hat Enterprise Linux

### Server operating system version

9.5

### What browsers are you using?

Firefox

**Comment by martinpitt** [comment]
Hello @logonoff ! It seems to me that this is actually intended, as per this comment: https://github.com/cockpit-project/cockpit/blob/ab76818721182151b/pkg/static/login.scss#L1027

It also seems to make sense to me, as the whole point of dark mode is to avoid bright colors in dark environments -- so tinting the logo would make sense, too?

@garrett can you please comment? The fix in #21379 is straightforward, but I'm not at all sure that we actually want it.

Thanks!

**Comment by logonoff** [comment]
Ah, if it's a feature feel free to disregard the PR. The darkened branding just looked a little off to my eyes and it probably violates some [brand guidelines](https://www.redhat.com/en/about/brand/standards/logo)

**Comment by garrett** [comment]
Yeah, it was intentional. We could either drop it completely, or we could split the difference and perhaps 75% or 80% instead of 66%? It does feel a little too dark.

https://github.com/cockpit-project/cockpit/blob/ab76818721182151b/pkg/static/login.scss#L1032C1-L1032C19

**Comment by logonoff** [comment]
Addressed in PF6 upgrade
